### ISSUE TITLE
[@types/mapbox-gl] - Update to 1.12.0

### ISSUE THREAD

**Issue by JeffBaumgardt** [DefinitelyTyped/DefinitelyTyped#46848]
Is something wrong with CI or publishing infrastructure?

- Check the [Infrastructure status updates](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44317) thread for known issues.
- Leave a message on the [DefinitelyTyped Gitter](http://gitter.im/DefinitelyTyped/DefinitelyTyped).

If you know how to fix the issue, make a pull request instead.

- [X] I tried using the `@types/xxxx` package and had problems.
- [X] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [X] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @dmytro-gokun

If you do not mention the authors the issue will be ignored.

Sorry for the issue/ping. Any updates for `mapbox-gl@1.12.0`?

**Comment by JeffBaumgardt** [comment]
Sorry missed a few authors
@dobrud
@patrickr
@macob
@LiamAttClarke
@life777
@mklopets
@amxfonseca

**Comment by dmytro-gokun** [comment]
@JeffBaumgardt 

> Sorry for the issue/ping. Any updates for mapbox-gl@1.12.0?

No. I do not get paid for this. I only work on this when i need it for my project or when i have time. At the moment neither of those conditions is met. So, you are welcome to create a PR. If you do not want to do that, you have to wait until someone does.

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
